my-vault/handlers/handlers.go

```diff
@@ -0,0 +1,164 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"github.com/google/uuid"
+	"sync"
+)
+
+type Secret struct {
+	Id    string 
+	Name  string  
+	KVMap map[string]string 
+	
+}
+var mu sync.Mutex
+var db = make(map[string]Secret)
+
+
+type CreateSecretRequest  struct {
+	Name  string    `json:"name"`
+	KVMap map[string]string `json:"kv_map"`
+	
+}
+
+type CreateSecretResponse  struct {
+	Message string `json:"message"`
+	Id string `json:"id"`
+	Name  string    `json:"name"`
+	
+	
+}
+type GetSecretResponse struct {
+	Message string `json:"message"`
+	Id    string  `json:"id"`
+	Name  string    `json:"name"`
+	KVMap map[string]string `json:"kv_map"`
+
+}
+
+func GenerateUUID() string{
+	newUUID, err := uuid.NewV7()
+	if err != nil {
+		fmt.Println("Error generating UUID:", err)
+		panic(err)
+	}
```

Comment:
I think it'd be better to return an error to the caller so they can handle or ignore it instead of panicing, what do you think?